developer-docs-site/docs/concepts/governance.md

```diff
@@ -33,9 +33,9 @@ sources={{
 
 ## Who can propose
 
-- To either propose or vote, you must stake but you are not required to run a validator node. However, we recommend that you be a validator with a stake and be a part of the validator set. 
+- To either propose or vote, you must stake, but you are not required to run a validator node. However, we recommend that you be a validator with a stake and be a part of the validator set. 
```

Comment:
nit: Rationale for why becoming a validator is recommended is that users will also gain rewards.